queries.sql

```diff
@@ -8,3 +8,54 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.50;
 SELECT * FROM animals WHERE neutered = TRUE;
 SELECT * FROM animals WHERE name NOT IN ('Gabumon');
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.40 and 17.30;
+
+BEGIN;
+UPDATE animals
+SET species = 'unspecified';
```

Comment:
- [ ] From your screenshots in the PR description, there is no screenshot showing the change of species column to UNSPECIFIED. Kindly run the query, do the screenshot and add to the PR.